scripts/bump-version.py

```diff
@@ -52,11 +53,10 @@ def postbump():
     APP_JUPYTERLITE_JSON.write_text(json.dumps(jupyterlite_json), **ENC)
     run(f"yarn prettier --write {APP_JUPYTERLITE_JSON}")
 
-    # save the new version to the top-level package.json
-    root_json = json.loads(ROOT_PACKAGE_JSON.read_text(**ENC))
-    root_json["version"] = py_version
-    ROOT_PACKAGE_JSON.write_text(json.dumps(root_json), **ENC)
-    run(f"yarn prettier --write {ROOT_PACKAGE_JSON}")
+    # save the new version to the top-level pyproject.toml
+    root_pyproject = toml.load(ROOT_PYPROJECT_TOML)
```

Comment:
A follow-up would be to make the releaser more resilient to repos using multiple top-levels files such as `package.json` and `pyproject.toml`.